### DISCUSSION TITLE
Welcome to UN Smart Maps Discussions!

### DISCUSSION THREAD

**Discussion by hfu** [UNopenGIS/7#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yuiseki** [comment]
Hi, my name is Yui Matsumura, please call me yuiseki.

Until a few years ago, I worked as a software engineer for an IoT startup company in Japan.
Today, I work as a product manager of large scale SaaS for a Japanese venture company.

My involvement with GIS goes back to my school days.
At the time, I was researching how GIS affects people's cognition, decision-making, behavior, and activities.

In the past, I have contributed to `sinsai.info` project during the 東日本大震災 and learned the importance of GIS in a disaster situation.
Nowadays, I am interested in making and self-hosting a vector tile map and utilize them in a disaster situation, as one of resource-limited environments.
Related to the above, I am also interested in OpenStreetMap project and community.

So I have a small contribution to `unvt/charites` and `unvt/portable`.
Otherwise, I have personally launch several projects about disaster news map, vector tile map and OpenStreetMap.

In light of the above, I would like to contribute to the following objectives in UN Open GIS Initiative DWG7:
- Objective 1: UN operation
- Objective 2: GSI operation
- Objective 3: UNVT Portable

I am looking for what I can and should do for each objective and project on every day.
If I can be of help for any other objectives than those listed above, I would love to do so.

That's all for my self-introduction. Best regards!

**Reply by hfu** [reply]
@yuiseki, thanks for sharing your story. Let's combine our efforts for good. 
Also thank you for your idea about creating discussion channels under this repository. 

I remember that I was so relieved to know that `sinsai.info` was there just after the Great East Japan Earthquake. I thought someone had to deploy Ushahidi here in Japan, but I was not able to do that. My contribution was to process aerial photographs as a member of the Technology Development Division of the Geospatial Information Authority of Japan (GSI).

As already announced before the Earthquake, I moved to the Overseas Project Division of the Ministry of Land, Infrastructure, Transport, and Tourism. My contribution there was to update the English version of the situation report. 

That was one of the backgrounds why we need a reliable open-source toolkit for geospatial information support for better situational awareness and operational decision-making. 

I am so proud to have you here in contribution to the UN Open GIS Initiative.

**Reply by ubukawa** [reply]
Hello. My name is Taro Ubukawa. Please call me @ubukawa in GitHub.
It is nice to meet you @yuiseki , @hfu  and all others.

I am now working for the deployment of open source vector tile in the United Nations.

My educational back ground is geology, and I have worked for GIS data development, etc since I started working in 2006.
I am not so good at developing a new UNVT tool as a developer, but I am actually using many of UNVT for my objective as a user.
Taking various opportunities in the future, I would like to share our experience and would like to find any possible collaboration with our colleagues here. 

our experience

- data production (about 150GB vector tile from PostGIS database. Regular update. and global RGB elevation tile development from NASADEM)
- data styling (developed a style for our own data, using unvt/charites)
- data hosting (nodejs/express server, a vector tile interface with Esri GeoPortal based on ArcGIS REST API, Azure AD authentication)
- data use (deployed several maplibre plug-ins for our map. tried 3D terrain view with MapLibre.)
- "core" related activities (use of docker image unvt/nanban, unvt/rgbify, etc. --> I would like to try WSL in the future)
- conducting vector tile practice to UN colleagues.

Overview of our work in UN (a presentation at a certain meeting):
https://speakerdeck.com/ubukawa/2021-12-16-unvt-unitemapconference

I look forward to working with you from now under the DWG7.

Best,
